src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -26,6 +26,8 @@ import LoggedInDetailPage from 'components/LoggedInDetailPage'
 import NotFound from 'components/NotFound'
 import Back from 'components/Back'
 import { Props, State } from './ItemDetailPage.types'
+import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
```

Comment:
Would you mind movings this import up so the relative imports are down below?
```suggestion
import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
import { Props, State } from './ItemDetailPage.types'
```